zk_toolbox/crates/zk_inception/src/commands/contract_verifier/init.rs

```diff
@@ -0,0 +1,45 @@
+use std::path::PathBuf;
+
+use common::{cmd::Cmd, logger, spinner::Spinner};
+use config::EcosystemConfig;
+use xshell::{cmd, Shell};
+
+use super::args::init::InitContractVerifierArgs;
+use crate::messages::{msg_binary_already_exists, msg_downloading_binary_spinner};
+
+pub(crate) async fn run(shell: &Shell, args: InitContractVerifierArgs) -> anyhow::Result<()> {
+    let args = args.fill_values_with_prompt(shell)?;
+    let ecosystem = EcosystemConfig::from_file(shell)?;
+    let link_to_code = ecosystem.link_to_code;
+    let zksolc_path = link_to_code
+        .join("etc/zksolc-bin/")
+        .join(args.zksolc_version.version);
+    let zkvyper_path = link_to_code
+        .join("etc/zkvyper-bin/")
+        .join(args.zkvyper_version.version);
+    let vyper_path = link_to_code
+        .join("etc/vyper-bin/")
+        .join(args.vyper_version.version.replace("v", ""));
```

Comment:
is it required to be inside the repo?

Reply:
Yes, the contract verifier uses the binaries inside the repo:
https://github.com/matter-labs/zksync-era/blob/main/core/bin/contract-verifier/src/main.rs#L21